baggage/templates/include/field.html

```diff
@@ -0,0 +1,4 @@
+{% if value %}
+    <dt>{{ desc }} {% if showlength %} ({{ value | length}}) {% endif %}</dt>
```

Comment:
You should be able to use the one already available instead of copying it.

Reply:
Same on meals, I think.

Reply:
Yup

Reply:
Removed.